CorsixTH/Lua/dialogs/menu.lua

```diff
@@ -657,7 +657,10 @@ function UIMenuBar:makeGameMenu(app)
   -- Edge Scrolling
   options:appendCheckItem(_S.menu_options.edge_scrolling,
     not app.config.prevent_edge_scrolling,
-    function(item) app.config.prevent_edge_scrolling = not item.checked end,
+    function(item)
+      app.config.prevent_edge_scrolling = not item.checked
+      app:saveConfig()
+    end,
```

Comment:
While this change seems to follow an existing pattern so it's fine here, the whole approach here looks wrong.

The better spot for saving would be on closing the window, or even just before ending the program, imho.